gems/laa_multi_step_forms/config/locales/en/layouts.yml

```diff
@@ -0,0 +1,5 @@
+---
+en:
+  layouts:
+    phase_banner:
+      phase: BETA
```

Comment:
Likewise the new v5 convention is for `Beta` rather than `BETA` for the phase banner.